orte/mca/ess/pmi/ess_pmi_module.c

```diff
@@ -203,7 +203,7 @@ static int rte_init(void)
     /* get the number of local peers - required for wireup of
      * shared memory BTL */
     OPAL_MODEX_RECV_VALUE(ret, OPAL_PMIX_LOCAL_SIZE,
-                          ORTE_PROC_MY_NAME, &u32ptr, OPAL_UINT32);
+                          &name, &u32ptr, OPAL_UINT32);
```

Comment:
But it isn't the wildcard name - and that could just as easily confuse someone!

Maybe call it something that indicates "wildcard_rank"?